battenberg/core.py

```diff
@@ -54,12 +56,18 @@ def _fetch_remote_template(self):
     def _cookiecut(self, cookiecutter_kwargs: dict, worktree: TemporaryWorktree):
         with tempfile.TemporaryDirectory() as tmpdir:
             logger.debug(f"Cookiecutting {cookiecutter_kwargs['template']} into {tmpdir}")
-            cookiecutter(
-                replay=False,
-                overwrite_if_exists=True,
-                output_dir=tmpdir,
-                **cookiecutter_kwargs
-            )
+            try:
+                cookiecutter(
+                    replay=False,
+                    overwrite_if_exists=True,
+                    output_dir=tmpdir,
+                    **cookiecutter_kwargs
+                )
+            except FailedHookException as e:
+                raise
+                # Suppress stacktrace for known hook error to ensure it is easy for user to diget.
+                logging.error(e)
```

Comment:
is this ever going to be hit due to the above raise on L67?

Reply:
I knew I'd forgotten something, looks like I missed that (was doing some debugging) and looks like I missed a test case too as all the tests passed! Will add that in, thanks for catching!